site/resources.md

```diff
@@ -99,9 +99,8 @@ Resources
   _add description_
 
 ### Visualization
-* Cartopy (basic)
-
-  _add description_
+* [Cartopy (basic)](https://nbviewer.jupyter.org/github/Unidata/pyaos-ams-2021/blob/master/notebooks/visualization/Cartopy-Intro.ipynb)
```

Comment:
This is close, it explicitly needs two spaces after the link (next to the header), then the new line -- markdown is very picky about these things.

Also, please bold the header: `* [**Cartopy (basic)**]`